examples/integrations/yolo/import_yolo_annotations.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>Replace the value of <code>API_KEY</code> with a valid [API key](<a href="https://docs.labelbox.com/reference/create-api-key)" target="_blank">https://docs.labelbox.com/reference/create-api-key)</a> to connect to the Labelbox client. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done